### PR TITLE
Compatibility with MC

### PR DESCRIPTION
**Pull request by pdeperio**
@tunnell can we make this an executable and/or copy to bin?

Several hacks to make lax work on MC.  Cut developers should think about this, and if their cut can't be used with MC, then add the appropriate exceptions here.

### CODE REVIEW

**Comment by pdeperio** [comment]
possible to add another flag for each cut indicating whether it's compatible with MC, so that here we just need to pass a single flag instead?

**Comment by pdeperio** [comment]
Requires https://github.com/XENON1T/hax/issues/47 to be fixed

File: examples/lax_root_batch.py [hunk 1]
```diff
@@ -8,38 +16,88 @@
 
 import root_pandas
 
+
 parser = argparse.ArgumentParser(description="Create lichen ROOT files with lax")
 
 parser.add_argument('--run_number', dest='RUN_NUMBER',
                     action='store', required=True, type=int,
-                    help='Run number to process')
+                    help='Run number to process (-1 for MC)')
```
Comment (tunnell) [inline examples/lax_root_batch.py]: Might want it just to be negative run number for MC.  E.g. -1402 would be SR0 run 1402 in MC etc.  But ok.

File: examples/lax_root_batch.py [hunk 2]
```diff
@@ -8,38 +16,88 @@
...
+TREENAME = 'tree'
+
+# MC
+if args.RUN_NUMBER < 0:
+
+    # No run dependent sims yet
+    PAX_VERSION_POLICY = 'loose'
+
+    # Use filename instead of run number
+    RUN_NUMBER = args.MINITREE_PATH+'/'+args.FILENAME
```
Comment (tunnell) [inline examples/lax_root_batch.py]: os.path.join

Comment (tunnell) [reply examples/lax_root_batch.py]: and spaces

File: examples/lax_root_batch.py [hunk 3]
```diff
@@ -8,38 +16,88 @@
...
+
+    # Remove meaningless variables
+    MINITREE_NAMES.remove('TailCut')
+    MINITREE_NAMES.remove('Proximity')
+
+    # Remove meaningless cuts (HARDCODE WARNING)
```
Comment (tunnell) [inline examples/lax_root_batch.py]: Remove by name.  There is a getname function.

File: examples/lax_root_batch.py [hunk 4]
```diff
@@ -8,38 +16,88 @@
...
+
+# Be careful what you're doing here
+if args.RUN_NUMBER < 0:
+    HAX_KWARGS['blinding_cut'] = 'run_number<=0'
 
-DF = hax.minitrees.load(args.RUN_NUMBER,
-                        ['Fundamentals', 'Basics', 'TotalProperties',
-                         'Extended', 'Proximity', 'TailCut'],
-)
+#HAX_KWARGS['runs_url'] = 'mongodb://eb:{password}@xenon1t-daq.lngs.infn.it:27017/run'
```
Comment (tunnell) [inline examples/lax_root_batch.py]: remove

**Review by tunnell — CHANGES_REQUESTED** [review]
Looks more or less fine, just small fixes

**Comment by tunnell** [comment]
Disabled travis for future too.

File: examples/lax_root_batch.py (hunk 1 above)
Comment (pdeperio) [reply examples/lax_root_batch.py]: Agree.  But until we get run dependence working at G4/emission/fax/pax level, this doesn't matter.

File: examples/lax_root_batch.py (hunk 2 above)
Comment (pdeperio) [reply examples/lax_root_batch.py]: This is changed in the [new bin script](https://github.com/XENON1T/lax/blob/d95be1f6fd2f230c2e26fd95ae0724e5c8eaf618/bin/laxer) (this example script will be removed).

File: examples/lax_root_batch.py (hunk 3 above)
Comment (pdeperio) [reply examples/lax_root_batch.py]: Don't understand what you mean here.  Can you write a snippet?

Comment (pdeperio) [reply examples/lax_root_batch.py]: N/m, fixed [here](https://github.com/XENON1T/lax/pull/57/commits/710f2906c3b1f9c053a1581311c8f5f69e354234#diff-e9f87b2a3f5a52d944d1bd9d5651c367R82)

**Review by tunnell — APPROVED** [review]

**Pull request merged**